compiler/plugins/target/ROCM/test/emit_debug_info.mlir

```diff
@@ -0,0 +1,52 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: iree-compile --iree-hal-target-backends=rocm --iree-hip-target=gfx942 \
```

Comment:
We prefer using iree-opt in tests and running the relevant passes (or pipelines) only